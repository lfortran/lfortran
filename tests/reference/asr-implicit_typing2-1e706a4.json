{
    "basename": "asr-implicit_typing2-1e706a4",
    "cmd": "lfortran --show-asr --implicit-typing --no-color {infile} -o {outfile}",
    "infile": "tests/errors/implicit_typing2.f90",
    "infile_hash": "36b4bb1882db636cc72a4e0b458d197bd21563dac2e0aa0640803ca9",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-implicit_typing2-1e706a4.stderr",
    "stderr_hash": "5a5d770bfc817e1cddf9a5567f3b695bcb88dffa5b3ffb694c57df41",
    "returncode": 2
}