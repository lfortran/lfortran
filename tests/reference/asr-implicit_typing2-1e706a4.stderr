semantic error: Variable 'x' is not declared
 --> tests/errors/implicit_typing2.f90:6:5
  |
6 |     x = 1
  |     ^ 'x' is undeclared
